pkg/kubelet/kubelet.go

```diff
@@ -3563,3 +3565,8 @@ func (kl *Kubelet) ListenAndServe(address net.IP, port uint, tlsOptions *server.
 func (kl *Kubelet) ListenAndServeReadOnly(address net.IP, port uint) {
 	server.ListenAndServeKubeletReadOnlyServer(kl, kl.resourceAnalyzer, address, port, kl.containerRuntime)
 }
+
+func isSyncPodWorthy(event *pleg.PodLifecycleEvent) bool {
+	// Filter out events that are not worthy of pod syncing
```

Comment:
nit: move this to the line above the function.
another nit: add a comment saying why we don't sync the pod on this event.